ci-operator/step-registry/baremetalds/sno/rename/baremetalds-sno-rename-commands.sh

```diff
@@ -308,15 +337,34 @@ do
 done
 
 oc adm wait-for-stable-cluster --minimum-stable-period=5m --timeout=30m
+
+declare -a components=(
+  "openshift-etcd-operator etcd-operator"
+  "openshift-kube-apiserver-operator kube-apiserver-operator"
+  "openshift-kube-controller-manager-operator kube-controller-manager-operator"
+  "openshift-kube-scheduler-operator openshift-kube-scheduler-operator"
+)
+for component in "${components[@]}"
+do
+  read -a tuple <<< "${component}"
+  namespace="${tuple[0]}"
+  app="${tuple[1]}"
+
+  if oc logs --namespace "${namespace}" --selector app="${app}" --tail=-1 |grep --quiet "RevisionTriggered"
```

Comment:
how can you tell that you are getting the logs after recert completed?

Reply:
The scripts wait first for recert to be completed [here](https://github.com/openshift/release/pull/48207/files#diff-a925b502a3b0c057a42c58f40252cab6b1bf3a04a09c2ba9304141150aeac406R323-R328). Is that what you are looking for?